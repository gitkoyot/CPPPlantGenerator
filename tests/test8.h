#ifndef TEST8_H
#define TEST8_H
#include "test7.h"

namespace test8WrapperNamespace
{


class test8
{
public:
	test8();
	~test8();
private:
	MainClassWithDependencyToStd classComposition;
	/* data */
}; /* -----  end of class test8  ----- */

}
#endif






