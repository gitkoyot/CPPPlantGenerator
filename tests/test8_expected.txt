@startuml
namespace test8WrapperNamespace #DDDDDD
class test8{
void test8 ( )
void ~test8 ( )
classComposition   :   MainClassWithDependencyToStd
}
end namespace 

test8WrapperNamespace.test8 *-- MainClassWithDependencyToStd
@enduml

